src/interfaces.ts

```diff
@@ -66,6 +66,8 @@ export interface PostGraphileOptions<
   ownerConnectionString?: string;
   // Enable GraphQL websocket transport support for subscriptions (you still need a subscriptions plugin currently)
   subscriptions?: boolean;
+  // ENable GrqphQL websocket transport support for query & Mutation
```

Comment:
This looks like steganography!
```suggestion
  // Enable GraphQL websocket transport support for query & mutation
```